buildSrc/src/main/kotlin/pmd-settings.gradle.kts

```diff
@@ -24,14 +24,14 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import io.spine.internal.dependency.Pmd
-
 plugins {
     pmd
 }
 
 pmd {
-    toolVersion = Pmd.version
+    // `Pmd.kt` is not migrated.
+    // A TOML file can't contain just a version.
```

Comment:
Why do we refer to the `Pmd` object here? E.g. instead of using it? Or, if we don't need the `Pmd` object any longer, why not just delete it in this PR?